src/index.js

```diff
@@ -1,13 +1,129 @@
+import {
+  BALL,
+  STRIKE,
+  NOTHING,
+  ERROR,
+  CORRECT,
+  RESTART,
+  RESTART_BTN,
+} from './define.js';
+
+const baseballGame = new BaseballGame();
+const computerInputNumbers = generateRandomNumbers();
+const userInputNumbers = document.getElementById('user-input');
+const divResult = document.getElementById('result');
+const buttonSubmit = document.getElementById('submit');
+
 export default function BaseballGame() {
   this.play = function (computerInputNumbers, userInputNumbers) {
-    return "결과 값 String";
+    const result = startGame(computerInputNumbers, userInputNumbers);
+
+    divResult.innerHTML = `${result}`;
+    if (result === CORRECT) {
+      // 정답이라면 restart버튼 추가
+      divResult.appendChild(createRestartButton());
+      const restart = document.getElementById('restart');
+      restart.addEventListener('click', removeResultElement);
+    }
+    return result;
   };
 }
 
-// export default class BaseballGame {
-//   play(computerInputNumbers, userInputNumbers) {
-//     return "결과 값 String";
-//   }
-// }
+buttonSubmit.addEventListener('click', () => {
+  if (checkValidInput(userInputNumbers) == false) {
```

Comment:
```suggestion
  if (checkValidInput(userInputNumbers) === false) {
```

자바스크립트에서 == 는 값을 비교하고, ===는 타입을 기반으로 비교하기 때문에 엄격한 비교인 ===를 항상 쓰는게 조금 더 좋아보여서 리뷰했습니다.

Reply:
형변환이 일어나면 안되는 곳에는 ===를 사용했는데 불리언 타입에도 그래야하는군요. 
오늘 아래 링크를 봤었습니다.
https://hyunseob.github.io/2015/07/30/diffrence-between-equality-and-identity-in-javascript/

Reply:
자바스크립트에서 == 는 오류의 소지가 많아서 저는 모든 경우 ===만 사용합니다.